kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -118,15 +121,26 @@
 
       fetchContentCollection(message) {
         const { options } = message;
+        const { activityType, onlyContent } = options;
+
+        let kinds = undefined;
+        if (activityType) {
+          kinds = activityType.join(',');
```

Comment:
There is no need to join here. The serializer takes care of it.

Reply:
Also I don't understand very well when the activity type (like Watch) gets converted to the kind (like Video), or if the names are used interchangable. There is a map here https://github.com/learningequality/kolibri/blob/6903afec0ca3ebdf65fbf0eb96ef5348ed9fc2d2/kolibri/core/content/api.py#L245-L250

Reply:
I didn't know about the serializers, in that case I've updated the PR to remove the three `join` that I used there.